release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4400,6 +4400,28 @@ $ oc adm release info 4.13.51 --pullspecs
 
 * Previously, Open vSwitch (OVS) could function by using older configurations after a cluster upgrade operations. These older configurations might have caused issues for networking components that interact with OVS. With this release, OVS is restarted after an upgrade operation so that OVS is forced to receive the latest configurations. (link:https://issues.redhat.com/browse/OCPBUGS-41723[*OCPBUGS-41723*])
 
+[id="ocp-4-13-52-bug-fixes"]
```

Comment:
This bug should be moved after line 4427 so it belongs in the 4.13.52 release section.